SearchRotatedArray.java

```diff
@@ -0,0 +1,43 @@
+// Time complexity : O(log n)
```

Comment:
what is n?

Reply:
n is size of the array and we are diving the array to search and element in one half of the array .
Is my understanding correct ?

Reply:
you have to mention like 
TC is O(log n) where n is the length of the nums array